circle.yml

```diff
@@ -1,17 +1,15 @@
 dependencies:
   override:
-    - make setup
+    - rm -rf node_modules
+    - case $CIRCLE_NODE_INDEX in 0) make setup ;; 1) nvm exec 0.12.7 make setup ;; 2) nvm exec 4 make setup ;; 3) nvm exec 5 make setup ;; 4) nvm exec 6 make setup ;; esac:
+        parallel: true
```

Comment:
According to the Node.js Foundation [user survey](https://nodejs.org/static/documents/2016-survey-report.pdf) only 18% are on Node 0.12 so I'm happy with dropping it.